validator/src/main.rs

```diff
@@ -1421,6 +1425,13 @@ pub fn main() {
                 .conflicts_with("no_accounts_db_caching")
                 .hidden(true)
         )
+        .arg(
+            Arg::with_name("disable_duplicate_instance_check")
+                .long("disable-duplicate-instance-check")
```

Comment:
nit: We use the "no-" prefix as a pattern for flags like this:
```suggestion
            Arg::with_name("no_duplicate_instance_check")
                .long("no-duplicate-instance-check")
```

Reply:
done